tuf/ngclient/_internal/trusted_metadata_set.py

```diff
@@ -336,6 +349,11 @@ def update_delegated_targets(
         if self.snapshot is None:
             raise RuntimeError("Cannot load targets before snapshot")
 
```

Comment:
```suggestion

        # client workflow 5.5.4 and 5.5.6
```
Maybe it will be even clearer if we add the steps numbers of the client workflow?